Misc/NEWS.d/next/Library/2024-10-02-21-11-18.gh-issue-124917.Lnwh5b.rst

```diff
@@ -0,0 +1,2 @@
+Allow calling :func:`os.path.exists` and :func:`os.path.lexists` with
+keyword arguments on Windows. Fixes a regression in 3.13.0.
```

Comment:
@nineteendo do you know in which version of 3.12 this would have regressed?

Reply:
3.12.4: https://docs.python.org/3.12/whatsnew/changelog.html#python-3-12-4-final

> [gh-118507](https://github.com/python/cpython/issues/118507): Fix [os.path.isfile()](https://docs.python.org/3.12/library/os.path.html#os.path.isfile) on Windows for pipes.